modules/standard/IO.chpl

```diff
@@ -4714,403 +5206,449 @@ proc _channel.writeIt(const x) throws {
   }
 }
 
-  /* Explicit call for reading or writing a literal as an
-     alternative to using :type:`IO.ioLiteral`.
-   */
-  inline
-  proc _channel.readWriteLiteral(lit:string, ignoreWhiteSpace=true) throws
-  {
-    var iolit = new ioLiteral(lit:string, ignoreWhiteSpace);
-    if this.writing then
-      this.writeIt(iolit);
-    else
-      this.readIt(iolit);
-  }
+/* Explicit call for reading or writing a literal as an
+   alternative to using :type:`IO.ioLiteral`.
+*/
+inline
+proc fileReader.readWriteLiteral(lit:string, ignoreWhiteSpace=true) throws
```

Comment:
Hmm I am not remembering discussing these readWriteLiteral methods. IMO it'd be reasonable to deprecate them or mark them unstable but that doesn't have to happen in this PR.

Reply:
Yeah, I also think they should be deprecated, but was worried doing so would hide what was happening because it's such a large PR already